Add intros (#6414)
github/docs: content/github/setting-up-and-managing-your-github-user-account/index.md

---
title: Setting up and managing your GitHub user account

> intro: Learn how to manage your GitHub user account settings, email preferences, collaborator access for personal repositories and organization memberships.

Reviewer: ```suggestion
intro: You can manage settings in your GitHub user account including email preferences, collaborator access for personal repositories, and organization memberships.
```
I updated this one to use the `You can` language, as well. I also tweaked it a little bit so we can use the word `including` which keeps this section more evergreen 🌲 That way if we add things to this category later, this intro will still be accurate!
